src/fork_gc.c

```diff
@@ -1243,10 +1243,35 @@ FGCError FGC_parentHandleFromChild(ForkGC *gc) {
   return status;
 }
 
+static inline bool isOutOfMemory(RedisModuleCtx *ctx) {
+  #define MIN_NOT_0(a,b) (((a)&&(b))?MIN((a),(b)):MAX((a),(b)))
+  RedisModuleServerInfoData *info = RedisModule_GetServerInfo(ctx, "memory");
```

Comment:
Did we verify that all of these APIs are safe to call without the GIL?

Reply:
Not safe, moving call to after taking the GIL and adding comment to function.